src/main/java/org/opentripplanner/graph_builder/GraphBuilder.java

```diff
@@ -74,6 +76,7 @@ public static GraphBuilder create(
     boolean hasTransitData = hasGtfs || hasNetex;
 
     GraphBuilder graphBuilder = new GraphBuilder(baseGraph);
+    graphBuilder.hasTransitData = hasTransitData;
```

Comment:
Since this is now becoming a public field and we are checking later if it really has data, how about calling this `hasTransitDatasets`?